src/librustc_error_codes/error_codes/E0751.md

```diff
@@ -5,8 +5,8 @@ Erroneous code example:
 ```compile_fail,E0751
 trait MyTrait {}
 impl MyTrait for i32 { }
-impl !MyTrait for i32 { }
+impl !MyTrait for i32 { } // error!
 ```
 
-Negative implementations are a promise that the trait will never be
-implemented for the given types.
+Negative implementations are a promise that the trait will never be implemented
+for the given types. Therefore, you can't have both at the same time!
```

Comment:
Should we use `can't`? Shouldn`t we use `cannot` since `can't` is usually used for informal or mouth-to-mouth conversation IIRC.

Reply:
Good point!